src/sdl_geometry.h

```diff
@@ -0,0 +1,58 @@
+#pragma once
+#ifndef CATA_SRC_SDL_GEOMETRY_H
+#define CATA_SRC_SDL_GEOMETRY_H
+
+#if defined(TILES)
+#include "memory.h"
```

Comment:
This is neither a game header nor a standard c/c++ header.

Reply:
Hol' up.. it is a [standard c++ header](https://en.cppreference.com/w/cpp/header/memory), right ? Or did you mean that I'm using quotes here and not angle brackets ?

Reply:
The c++ header is `memory` (without the extension), not `memory.h`.

Reply:
Oh, I thought `<memory>` is just a shortcut for `memory.h` within the system defined include directories, but it's actually an implementation-defined feature. Good to know !